features/step_definitions/events.rb

```diff
@@ -0,0 +1,21 @@
+Given(/^an event (.*)$/) do |eventName|
+  create_event_named eventName
+end
+
+Given(/^user (.*) is organizer of event (.*)$/) do |userName, eventName|
+  event = event_named eventName
+  create_user_named userName
+  user = user_named userName
+  event.editors << user
+  user
+end
+
+Then(/^(.*) should be listed as organizer$/) do |userName|
+  user = user_named userName
+  expect(page).to have_text(user.name)
```

Comment:
Does anyone have a better idea on how to specifically look for organizers? They are displayed as a list in a `description list`, so a simple `have_text: Organizer:...` won't work.

Reply:
fine for now imo.